lib/matplotlib/image.py

```diff
@@ -56,11 +56,67 @@
     'blackman': _image.BLACKMAN,
 }
 
-
 interpolations_names = set(six.iterkeys(_interpd_))
 
 
-class _AxesImageBase(martist.Artist, cm.ScalarMappable):
+def composite_images(images, renderer, magnification=1.0):
+    """
+    Composite a number of RGBA images into one.
+
+    Parameters
+    ----------
+    images : list of objects with `make_image` method
+
+    renderer : RendererBase instance
+
+    magnification : float
+        The additional magnification to apply for the renderer in use.
```

Comment:
Might also want to make it clear whose responsibility it will be to handle zorder.